app/src/main/java/br/org/cesar/discordtime/stickysessions/injectors/modules/SessionPresenterModule.java

```diff
@@ -0,0 +1,21 @@
+package br.org.cesar.discordtime.stickysessions.injectors.modules;
+
+import br.org.cesar.discordtime.stickysessions.domain.model.Note;
+import br.org.cesar.discordtime.stickysessions.domain.model.Session;
+import br.org.cesar.discordtime.stickysessions.executor.ObservableUseCase;
+import br.org.cesar.discordtime.stickysessions.presentation.session.SessionContract;
+import br.org.cesar.discordtime.stickysessions.presentation.session.SessionPresenter;
+import dagger.Module;
+import dagger.Provides;
+
+@Module
```

Comment:
Should All presenter modules, and all it's dependencies, be singletons? Since we only have a single presenter per view (Activity/Fragment)?